day1/ques1.rb

```diff
@@ -0,0 +1,21 @@
+# 1. wap to display number is prime or not
+
+def primeOrNot(num)
```

Comment:
Always prefer snake case in ruby method names - primeOrNot -> prime_or_not -> better name -> `is_prime_number?(number)`